Max9545/What-s-Cookin-Refactory
# Mbb/fix/pass user tests

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*

Corrected tests in user class test file and created check pantry method for both when a user does and does not have enough ingredients to complete the wanted recipe. If enough ingredients are present in user pantry then a string of success is returned, if the user does not have enough in their pantry a list in the form of an object telling the user what they need and how much of each item will be returned.

## *Instructions for reviewer:*
*Where should the reviewer start?*

user-test.js - reformat data for user, tests on 139 and 146 for checkPantry method

user.js - line 35 for new checkPantry 

*How should this be manually tested?*

run npm test and check user class

*Any background context you want to provide?*

Read all the tests!

*What are the relevant tickets?*

closes #2

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

Maybe come back and interpolate missing item list if need be

## Checklist:
